### PR TITLE
Fixed some of the dependency issues pointed out by dependabot

### PR DESCRIPTION
**Pull request by KevinCLydon**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2,21 +2,6 @@
 # It is not intended for manual editing.
 version = 3
 
-[[package]]
```
Comment (lbergelson) [inline Cargo.lock:5]: Did this previously depend on 2 different versions of actix-codec?  This file is autogenerated I assume?

**Review by lbergelson — APPROVED** [review]
This looks good to me.  Numbers went up instead of down.  What's going on with the 2 versions of actix-codec in the cargo.lock, can rust depend on both?

File: Cargo.lock (hunk 1 above)
Comment (KevinCLydon) [reply Cargo.lock:5]: Yeah, Cargo.lock is autogenerated.  By my understanding, multiple versions of a library can exist in the project if they're being used in different places?  In this case, two different libraries were using two different versions of actix-codec and one of those versions was bad.

Comment (lbergelson) [reply Cargo.lock:5]: That's neat that it can use both at once.  Java gets into trouble when that happens.

**Pull request merged**